exercises/hello_rubyric/config.yaml

```diff
@@ -0,0 +1,13 @@
+_external: true
+title: Hello Rubyric
+description: A simple grading example with Rubyric and Python
+max_submissions: 10
+max_points: 10
+min_group_size: 1
+max_group_size: 2
+points_to_pass: 6
+key: rubyric-python
```

Comment:
key: no key should be defined here since it is taken from the RST submit directive parameters. I guess the RST compilation overwrites this key that was manually defined in config.yaml. It is better to remove it from here since it is not used.

Reply:
Yes, you are right.